### PR TITLE
Removed Camper News Option

### PR DESCRIPTION
**Pull request by abisz**
Noticed that posting on camper news was still listed as an option for getting brownie points

### CODE REVIEW

**Comment by alayek** [comment]
## LGTM :+1: 

But I would still like @Rafase282 @raisedadead @abhisekp to take a look, because I want to be absolutely sure that there are only those two ways to get brownie.

**Comment by raisedadead** [comment]
Yup this is good to go!

File: Brownie-Points.md [hunk 1]
```diff
@@ -2,12 +2,11 @@ The number beside your picture on Free Code Camp tells you how many Brownie Poin
 
 ![A user profile picture next to a with Brownie Points score](http://i.imgur.com/SsvbkDH.png)
 
-There are three ways you can get Brownie Points:
+There are two ways you can get Brownie Points:
 
 1. Complete challenges - you get one point per challenge you complete    
-1. Post relevant links on [Camper News](Camper-News) - you get 1 point for posting a link, and another point for each upvote your link gets    
 1. Help other campers in chat - each time you help another camper and they thank you (by typing "thanks @yourname"), you will get a point
```
Comment (abhisekp) [inline Brownie-Points.md]: The markdown `1.` is parsed as `2`. But in the text it should be `2`.

Comment (abisz) [inline Brownie-Points.md]: @abhisekp I thought this was a way to write ordered lists in markdown...

Comment (abhisekp) [inline Brownie-Points.md]: @abisz sure. Indeed that's the way. Just the numbering in the markdown text can be written in incremental order. Although, markdown renderers will show it in proper order even if numbering is messed up but just for consistency, it is good to write it in the following way...

```md
1. Item 1
2. Item 2
3. Item 3
```

Ref. [**Markdown Guide**](https://guides.github.com/features/mastering-markdown/)

Comment (abhisekp) [inline Brownie-Points.md]: @abisz Would you like to fix the required changes?

Comment (abisz) [inline Brownie-Points.md]: @abhisekp I don't know, should I? It would need another PR, right?

Comment (alayek) [inline Brownie-Points.md]: Yes, @abisz. You need to update your fork first. Then raise another PR.

Comment (abhisekp) [inline Brownie-Points.md]: > ----
<p align="center">Need any help?<br>Read <a href="https://github.com/FreeCodeCamp/wiki/blob/master/.github/CONTRIBUTING.md"><b>CONTRIBUTING</b></a> Guidelines<br>Chat with us in <kbd><a href="https://gitter.im/FreeCodeCamp/Wiki"> <b>FreeCodeCamp/Wiki</b></a></kbd></p>

> ----

**Comment by abhisekp** [comment]
Thanks for your contribution. :tada: :smiley:

**Pull request merged**